website/templates/util/render_node.mako

```diff
@@ -143,13 +143,13 @@
         class="project list-group-item list-group-item-node">
         <p class="list-group-item-heading f-w-lg">
             %if summary['is_registration']:
-                Private Registration
+                Unshared Registration
```

Comment:
Why was this language changed? I don't see any logic that checks whether the user is viewing through a view-only link.